scripts/seed-db.ts

```diff
@@ -90,28 +111,24 @@ async function seedDatabase(): Promise<void> {
     roles.content
       .filter((role: RoleOutput) => !CONFIG.ROLES_TO_KEEP.includes(role.name))
       .map((role: RoleOutput) => {
-        return axiosInstance.delete(`/v1/roles/${role.id}`)
+        return axiosInstance.delete(`/v1/roles/${role.name}`)
       }),
   )
 
   // Create new roles and users
   console.log('➡ Creating new roles and users')
 
-  const { data: rights } = await axiosInstance.get('/v1/rights')
-
   const createdRolesResponse = await Promise.all(
     Array(CONFIG.NUM_ENTITIES)
       .fill(null)
-      .map(() => {
+      .map((_, index) => {
         return axiosInstance
           .post<RoleOutput>('/v1/roles', {
             description: faker.name.jobTitle(),
             editable: faker.datatype.boolean(),
-            name: faker.name.jobType().toUpperCase(),
-            protected: faker.datatype.boolean(),
-            rights: faker.helpers.arrayElements(
-              rights.content.map((right: RightOutput) => right.id),
-            ),
+            name: ROLES_TO_CREATE[index],
+            protected: false,
+            rights: [],
```

Comment:
Don't we want to link the roles to rights? I see that it's for simplicity reasons but isn't it more realistic if they are linked?

Reply:
Ah sorry, forgot the explanation for this line: this was the reason, why the script failed. It also created all basic rights for every role. So if we tried to delete a role which has associated rights, we can't. Unfortunately, the backend does not provide an endpoint to delete rights 🙁 So I decided to just create no new rights for a role but rather use the existing ones which are sufficient imo. The rights CRUD is pretty much backed in into the backend logic.